app/assets/stylesheets/components/_external-link.scss

```diff
@@ -5,7 +5,7 @@ $theme-external-link-sr-label-tab-same: "External.";
 $theme-external-link-sr-label-tab-new: "External, opens in a new tab";
 
 @mixin external-link($contrast-bg: "default") {
-  display: inline-block;
+  display: inline;
```

Comment:
this `inline-block` was causing links to not have line breaks and it looked weird. it will affect other pages that have external links but i couldn't find any examples of places where you wouldn't want the line to break